documentation/docs/internals/data-serialization.md

```diff
@@ -25,13 +25,29 @@ We try to support all JavaScript built-in types. Currently we support:
 * Objects (plain and class instances)
 * Collections (Array, Map and Set)
 * Data buffers (Typed Arrays)
-* Others (Date and URL)
-
-And we have planned to add support for:
-
 * Big integers
 * Regular expressions
+* Others (Date and URL)
```

Comment:
```suggestion
* Others (BigInt, Date, RegExp and URL)
```

Reply:
I doubted between my writings and your suggestion 😊